bindings/cpp/Cargo.toml

```diff
@@ -34,6 +34,8 @@ crate-type = ["staticlib"]
 anyhow = "1.0"
 chrono = "0.4"
 cxx = "1.0"
+# cxx-async v0.1.1 in crates.io has build problems, so we use git repo directly
+cxx-async = { git = "https://github.com/pcwalton/cxx-async", rev = "961dd106b8eb2a86991728e1a18948e597426c1a", optional = true }
```

Comment:
It appears that cxx-async is no longer maintained. Should we consider forking it and maintaining it ourselves?

Reply:
Yeah I also noticed that. Good to folk it so that we don't need to wait maintainers response while contributing.

Reply:
This may prevent us from merging this PR. OpenDAL does not permit dependencies on a git version crate, as this could disrupt our release process.

I'm trying to contact with @pcwalton at https://github.com/pcwalton/cxx-async/issues/6

Reply:
I see. Hope the author can respond so we don't need to folk.